### PR TITLE
Finish mobile version - Day 3 Project

### PR DESCRIPTION
**Pull request by Mujeeb4582**
Install linters files
Add index.html
Add style.css
Add work section
Add about section
Add form
Add contact section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -57,5 +57,232 @@
...
+					<h2 class="headline">About me</h2>
+					<div class="about">
+						<p>I can help you bulid a product, feature or website. Look through some of my work and experience! if you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+					</div>
+				</div>
+				<div class="grid-item3">
+					<button class="resume" type="button"><a href="#">Get My Resume</a></button>
+			</section>
+	
+				<!-- About section end -->
```
Comment (cwaku) [inline index.html]: - [ ] To ensure that we follow the design template as close as possible, I strongly suggest that you add the background images to this section as shown in the design.
- [ ] Also, Kindly keep an eye on the font sizes and line heights on the Figma design and replicate that.

Your Design | Figma Design
--|--
image | image

File: index.html [hunk 2]
```diff
@@ -57,5 +57,232 @@
...
+      </div>
+      <h3 class="Skills-title">Skills</h3>
+      <ul class="skills-list">
+        <li>Codekit</li>
+        <li>GitHub</li>
+        <li>Codepen</li>
+        <li>Gitlab</li>
+        <li>Terminal</li>
+      </ul>
+    </section>
```
Comment (cwaku) [inline index.html]: - [ ] The languages, frameworks and skills shouldn't be in their own section tags rather they should be inside of the "About me" section. Kindly fix that. 🙏🏾

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @Mujeeb4582 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
There are a few issues 🐛  that you still need to work 🛠️  on to go to the next project ⏭️, but you are almost there! 🏃🏾 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 

## Required Changes ♻️
It's ok to have bugs in your code. Please, see this code review as a chance to learn practical coding skills and new tips 🌟  that can improve your code quality 💯 and help you get a better job 💼.

I will guide 📔 you through fixing 🛠️ these bugs, follow my comments down below and make sure you don't miss any of them to get your project approved ✅

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alexpuente0 — APPROVED** [review]
Hi @Mujeeb4582 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions
- [OPTIONAL] Make sure that the background in the 'About me' section matches the alignment on the desin.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**